Added Ability Supersweet Syrup

FL- opened the pull request.

FL- committed:
Added Ability Supersweet Syrup

Maruno17 commented:
I assumed this should be coded like Intimidate, to ensure the messages are informative if the ability splash isn't being used.

FL- committed:
Updated Supersweet Syrup with some intimidate checks

Updated Supersweet Syrup with some intimidate checks (substitute and Contrary).

FL- committed:
Merge branch 'dev' into ability_supersweet_syrup

FL- commented:
> I assumed this should be coded like Intimidate, to ensure the messages are informative if the ability splash isn't being used.

Done!

Maruno17 commented:
This ability only triggers once per battle. Look at Intrepid Sword.

FL- committed:
Merge branch 'dev' into ability_supersweet_syrup

FL- committed:
Updated Supersweet Syrup to work only once

FL- commented:
> This ability only triggers once per battle. Look at Intrepid Sword.

Done!

Maruno17 committed:
Added missing part to Supersweet Syrup's effect

Maruno17 committed:
Minor tweak to Supersweet Syrup's OnSwitchIn handler
